style.css

```diff
@@ -1,46 +1,92 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
-  background-color: #fff;
-  width: 375px;
+  background-color: #e5e5e5;
   top: 0;
   font-family: "Poppins", sans-serif;
   border: 0;
   margin: 0;
   padding: 0;
-  background-image: url("Images/header-shapes-mobile.png");
-  background-repeat: no-repeat;
-  background-position-x: -0;
-  background-position-y: 100px;
 }
 
 li {
   list-style: none;
 }
 
 section {
-  display: flex;
-  flex-direction: column;
   position: relative;
-  top: 75px;
+  top: 48px;
   padding-left: 24px;
   padding-right: 24px;
-  margin-bottom: 20px;
+  font-family: "Poppins", sans-serif;
+  background-image: url("images/desktop-header.png");
```

Comment:
- [ ] Kindly fix the typo path for the background image, it will be `background-image: url("Images/desktop-header.png");` ,